packages/component/src/SendBox/SuggestedAction.js

```diff
@@ -28,12 +29,21 @@ const connectSuggestedAction = (...selectors) =>
     ...selectors
   );
 
-const SuggestedAction = ({ buttonText, click, disabled, image }) => {
+const SuggestedAction = ({ buttonText, clearSuggestedActions, disabled, displayText, image, text, type, value }) => {
   const [{ suggestedAction: suggestedActionStyleSet }] = useStyleSet();
+  const performCardAction = usePerformCardAction();
+
+  const handleClick = useCallback(() => {
+    performCardAction({ displayText, text, type, value });
+    type === 'openUrl' && clearSuggestedActions();
+
+    // TODO: Use the following line when setSuggestedActions hook is merged
+    // type === 'openUrl' && setSuggestedActions([]);
```

Comment:
I think this is what  I wanted to do originally... I can't remember exactly why you wanted to create a `clearSuggestedActions` method instead.

Reply:
The `clearSuggestedActions` function is in the original `connectToWebChat`.

Since we didn't implement the `useSetSuggestedActions` hook yet, we need to continue to use legacy `clearSuggestedActions` until the hook is implemented.